pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -348,10 +348,17 @@ func (b *volumeBinder) BindPodVolumes(assumedPod *v1.Pod) (err error) {
 		return err
 	}
 
-	return wait.Poll(time.Second, b.bindTimeout, func() (bool, error) {
+	if err = wait.Poll(time.Second, b.bindTimeout, func() (bool, error) {
```

Comment:
This is a long `if` that's hard to read.

I'd prefer to split it:

```
err = wait.Poll(...)
if err != nil {
```

Reply:
@jsafrane 
Can you take another look ?